docs/PAX_CONTROL_GROUPS.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pax Controla Groupiana
+category: Users, Groups and Home Directories
```

Comment:
IIUC, this will make the page show in the listing. I think we don't want t do this. Even though the page has a header that says that it's outdated, people might miss that. I think it's nice to move it into our docs tree, but I'd prefer to either omit it from the TOC or to add a separate "historical documents" section and link it from there.

@poettering?